common/scripted_effects/wc_horde_invasion_effects.txt

```diff
@@ -45,28 +45,93 @@ spawn_horde_troops_effect = {
 			inheritable = no
 			name = horde_event_troops
 			levies = {
-				value = 1250
+				value = 1000
 			}
 			men_at_arms = {
-				type = light_footmen
-				stacks = 3
+				type = savage
+				stacks = 1
 			}
 			men_at_arms = {
-				type = light_horsemen
+				type = blackrock_legionnaire
 				stacks = 1
 			}
 			men_at_arms = {
-				type = armored_horsemen
-				stacks = 1
+				type = wolfrider
+				stacks = 2
 			}
 			men_at_arms = {
-				type = mangonel
-				stacks = 1
+				type = trebuchet
+				stacks = 10
```

Comment:
now that's a strong change, from one mangonel to 10 trebuchets...

Reply:
> now that's a strong change, from one mangonel to 10 trebuchets...

The reason is said in the changelog.

> The Horde got more siege weapons to speed up the First and Second Wars.

Reply:
I know... Just, going from 0.3 (1x mangonel) to 4 (10x trebuchet) is a lot in my eyes...

Reply:
I don't get your question or suggestion then.

Reply:
It was more of a statement that it's a big change then a question/suggestion :D